examples/7SegmentLED_BasicUsage_ChangeOrderDigits

```diff
@@ -0,0 +1,96 @@
+// 
+// Copyright (c) 2019 smdn (https://smdn.jp/)
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#include <Arduino.h>
+#include <TM1637_7SegmentLEDController.h>
+
+#if ESP32
+#define PIN_DIO 16 // The pin number connected to TM1637's DIO (#17)
+#define PIN_CLK 17 // The pin number connected to TM1637's CLK (#18)
+#else
+#define PIN_DIO 6 // The pin number connected to TM1637's DIO (#17)
+#define PIN_CLK 7 // The pin number connected to TM1637's CLK (#18)
+#endif
+
```

Comment:
Could you please add some explanation about this?

+ What is the situations this is needed?
+ What is the difference of the behavior between the array specified or not?

I want to add explanation, but am not good at English as you see.
Can you write it as a reference?

Reply:
Yes I can certainly do that :)  Will do that today/tomorrow

Reply:
This is done